server/controllers/v1/post.ts

```diff
@@ -314,9 +339,36 @@ export default class PostController {
       };
     }
 
+    if (tags) {
+      // could not get adding (multiple) tags to work any other way
+      const tagObjs = await Promise.all(
+        tags.map(async (text) => {
+          return this.setupTag(text.trim());
+        })
+      );
+      await post.addTags(tagObjs);
+    }
+
     return { status: 200, data: { result: post } };
   }
 
+  /**
+   * Find, or create, a specific tag entry
+   *
+   * @param text  - The text to be in the tag
+   * @returns The tag object
+   */
+  async setupTag(text: string): Promise<Tag> {
+    // findOrCreate is broken when using sqlite, so I had to make this function.
+    // https://github.com/sequelize/sequelize/issues/5535
+    const data = await this.tagsRepo.findByPk(text);
+    return data
+      ? data
+      : await this.tagsRepo.create({
+          text,
+        });
```

Comment:
The issue seems to be related to a test environment only. If we had more time ideally we would update the test environment to use a non-in-memory db to fix it, rather than hack a solution around to get around the test config.

Reply:
Yeah.. if only :-(